src/calendar.js

```diff
@@ -58,6 +67,36 @@ const Day = React.createClass({
 const Calendar = React.createClass({
   displayName: 'Calendar',
 
+  getDate(i, w) {
+    const prevMonth = w === 0 && i > 7
+    const nextMonth = w >= 4 && i <= 14
+    const addMonth = prevMonth ? -1 : nextMonth ? 1 : 0
+    return this.props.moment.clone().add(addMonth, 'month').set('date', i).startOf('day')
+  },
+
+  isOther(i, w) {
```

Comment:
Can you update the README.md with the new range API and the fact that you can change colorSecondary.

Should also update the "This repo is forked from moment-input with the following changes" section.